modules/efk-logging-deploy-subscription.adoc

```diff
@@ -27,9 +27,152 @@ The initial set of {product-title} nodes might not be large enough to support th
 {product-title} cluster to run with the recommended or higher memory. Each Elasticsearch node can operate with a lower
 memory setting though this is not recommended for production deployments.
 
-. Create a project for cluster logging. You must create the project with the CLI:
 
-.. Create a YAML file with the following:
+.Procedure
+
+. Install the Elasticsearch Operator using the CLI to ensure the necessary values are set:
+
+.. Create a namespace for the Elasticsearch operator (for example, `eo-project.yaml`):
++
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-operators-redhat <1>
+  annotations:
+    openshift.io/node-selector: ""
+  labels:
+    openshift.io/cluster-logging: "true"
+    openshift.io/cluster-monitoring: "true"
+----
+<1> You must specify the `openshift-operators-redhat` namespace.
+
+.. Create the Namespace object:
++
+----
+$oc create -f eo-project.yaml
+----
+
+.. Create a Subscription object YAML file (for example, `eo-sub.yaml`) to
+subscribe a namespace to an Operator. 
++
+.Example Subscription
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  generateName: elasticsearch-
+  namespace: openshift-operators
+spec:
+  source: elasticsearch
+  sourceNamespace: openshift-operators-redhat <1>
+  name: elasticsearch-operator
+channel: preview
+----
+<1> You must specify the `openshift-operators-redhat` namespace.
+
+. Create the Subscription object:
++
+----
+$ oc create -f eo-sub.yaml
+----
+
+.. Create a CatalogSourceConfig object YAML file (for example, `eo.csg.yaml`) to enable the Elasticsearch Operator on the cluster. 
++
+.Example CatalogSourceConfig
+[source,yaml]
+----
+apiVersion: "operators.coreos.com/v1"
+kind: "CatalogSourceConfig"
+metadata:
+  name: "elasticsearch"
+  namespace: "openshift-marketplace"
+spec:
+  targetNamespace: "openshift-operators-redhat" <1>
+  packages: "elasticsearch-operator"
+----
+<1> You must specify the `openshift-operators-redhat` namespace.
++
+The Operator generates a CatalogSource from your CatalogSourceConfig in the
+namespace specified in `targetNamespace`.
+
+.. Create the CatalogSourceConfig object:
++
+----
+$ oc create -f eo.csg.yaml
+----
+
+.. Create an Operator Group object YAML file (for example, `eo-og.yaml`) for the Elasticsearch operator:
```

Comment:
This step should come before the catalogsourceconfig and subscription steps, and after creating the namespace.  At least, that's how it works in logging CI